apps/andi/lib/andi_web/live/edit_live_view.ex

```diff
@@ -359,6 +372,35 @@ defmodule AndiWeb.EditLiveView do
      )}
   end
 
+  def handle_info({:remove_data_dictionary_field_succeeded, deleted_field_parent_id, deleted_field_index}, socket) do
+    new_selected_field =
+      socket.assigns.changeset
+      |> get_new_selected_field(deleted_field_parent_id, deleted_field_index)
+
+    new_selected_field_id =
+      case new_selected_field do
```

Comment:
do we know if changing how this works affects that bug?

Reply:
It doesn't :(